raspimouse:
  ros__parameters:
    odometry_scale_left_wheel : 1.0
    odometry_scale_right_wheel: 1.0
    use_light_sensors         : true
    use_pulse_counters        : false
    wheel_diameter            : 0.048  # meters
    wheel_tread               : 0.0925  # meters
    pulses_per_revolution     : 400.0
    light_sensors_hz          : 100.0
    odom_hz                   : 100.0
    switches_hz               : 10.0
    initial_motor_power       : false
    odom_frame_id             : "odom"
    odom_child_frame_id       : "base_footprint"
    odom_frame_prefix         : ""
